### PR TITLE
modify new-innovations layout and style in HomePage

### PR DESCRIPTION
**Pull request by gregoriusjimmy**
## Type of changes

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Enhancement
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behaviour?

- Currently, new-innovations section only show one innovation per screen

## What is the new behavior?

- Make new-innovations section show all available new-innovations without a carousel
## Checklist

- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have filled the `[Unreleased]` [Changelog](https://github.com/nodefluxio/weber/blob/main/CHANGELOG.md

## Security Pentester API Checklist
- [ ] IDOR
- [ ] Malicious Script Injection
- [ ] SQL Injection

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

innovation1
innovation2
innovation5

### CODE REVIEW

**Review by lazuardyk — APPROVED** [review]
Work di local gw. Jadi lebih keren innovationnya daripada atasnya wkwkwk

**Review by sonadztux — APPROVED** [review]
Work juga di laptop gue, plus udah responsive gue buka langsung dari hp. Cool!

File: frontend/app/components/templates/HomePage/HomePage.module.scss [hunk 1]
```diff
@@ -251,42 +251,72 @@
...
+            .img {
+              border-radius: 1.5rem 0 0 1.5rem;
+              @include phone {
+                border-radius: 1.5rem 1.5rem 0 0;
+              }
+            }
+          }
+        }
+        display: flex;
+        background-color: #240a50;
```
Comment (williammfu) [inline frontend/app/components/templates/HomePage/HomePage.module.scss:274]: Kenapa `background-color`nya ga dibuat `$primary-color` aja? (Menurut gue lebih senada tapi jadi tenggelem sih tulisannya)

Comment (williammfu) [reply frontend/app/components/templates/HomePage/HomePage.module.scss:274]: `#240a50` lebih masuk warnanya

**Review by williammfu — APPROVED** [review]

**Pull request merged**